tests/driver.sh

```diff
@@ -357,4 +379,47 @@ int main()
 }
 EOF
 
+# format
+try_output 0 "2147483647" << EOF
```

Comment:
I wonder if here `0` is necessary or not. Can we eliminate it?

Reply:
Probably not. It is rare to have a test program to check both exit code and output. I'll remove it.

Reply:
How about providing both expected exit code and expected output and integrate them into function `try`.

Reply:
> How about providing both expected exit code and expected output and integrate them into function `try`.

I agree with this proposal. Please go ahead.